cluster/images/conformance/run_e2e.sh

```diff
@@ -39,23 +39,28 @@ saveResults() {
 # We get the TERM from kubernetes and handle it gracefully
 trap shutdown TERM
 
-ginkgo_args=(
-    "--focus=${E2E_FOCUS}"
-    "--skip=${E2E_SKIP}"
-    "--noColor=true"
-)
-
+ginkgo_args=()
 if [[ -n ${E2E_DRYRUN:-} ]]; then
     ginkgo_args+=("--dryRun=true")
 fi
 
 case ${E2E_PARALLEL} in
-    'y'|'Y')           ginkgo_args+=("--nodes=25") ;;
-    [1-9]|[1-9][0-9]*) ginkgo_args+=("--nodes=${E2E_PARALLEL}") ;;
```

Comment:
Removing this breaks `sonobuoy`'s `--e2e-parallel` flag which previously let users like me exercise our best judgement in choosing test execution parallelism. Now the only way to increase parallelism for me is to spin up a VM with larger vcpu count.